opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -556,19 +557,21 @@ namespace Opm {
                     }
                     for (unsigned phaseIdx = 0; phaseIdx < FluidSystem::numPhases; ++ phaseIdx) {
                         Scalar tmpSat = saturationsNew[phaseIdx] - saturationsOld[phaseIdx];
-                        resultDelta += tmpSat*tmpSat;
-                        resultDenom += saturationsNew[phaseIdx]*saturationsNew[phaseIdx];
+                        resultDeltaSaturation += tmpSat*tmpSat;
+                        resultDenomSaturation += saturationsNew[phaseIdx]*saturationsNew[phaseIdx];
```

Comment:
Very minor nit: The two [`assert()`s below](https://github.com/OPM/opm-simulators/blob/09122a9cb285d2024fc20dcb4c9108d0bffa3cfb/opm/simulators/flow/BlackoilModelEbos.hpp#L562-L563) continue to refer to `resultDelta` and `resultDenom` and that leads to [a build failure](https://ci.opm-project.org/job/opm-simulators-PR-builder/5324/console) in OPM's default build configuration (release mode, but with `assert()`s active).  Would you mind updating those statements too, please?

Reply:
> Done!

Much appreciated!  Thank you for the quick response and turn-around.

> (Sorry about the bug!)

Don't worry about it.  Anything the compiler can diagnose ahead of time is never a problem.